.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -30,115 +30,3 @@ jobs:
 
   - publish: build_artifacts/noarch/
     artifact: conda_pkgs_noarch
-
```

Comment:
You should not be making changes to this file.

Reply:
Hi @dopplershift , if I don't want to build any CUDA backend, should I revert the changes to this file and set some condition in my recipe?

Reply:
I'm not an expert on the compiled side here, but IIRC the CUDA builds are opt-in. The CI jobs that ran look like the standard ones to me. @conda-forge/help-c-cpp anything to add?

Reply:
I had the impression it was opt-out, but I think you're right. I've just reverted the changes.